### DISCUSSION TITLE
How to handle an OBJ file that's missing texture coordinate component?

### DISCUSSION THREAD

**Discussion by kizjig** [assimp/assimp#3628] [Q&A]
I just saw that discussions where enabled after I created my issue in git. 
https://github.com/assimp/assimp/issues/3627

So I'm going to ask my question here too.

I have an obj file that is exported without UVs from a 3rd party tool (CADSoftTools) and it doesn't contain all the face components for a face row in the raw data.

Eg:-
`f 2/1 3/1 1/1`

instead of looking like this (which is read fine by assimp):-
`f 2//1 3//1 1//1`

So normally the format for f command is `f position_id/texture_coordinates_id/normal_id`. In my example data above I have `f position_id/normal_id`

When it is attempted to be loaded by assimp I get the following error: "Failed to load file (Invalid face indice)"

**Is there currently anyway to load the file in assimp where it doesn't expect the uv data component and it would happily read 2 components per face indice instead of 3?**

**Comment by kimkulling** [comment]
Normally there are tests to avoid the describes situation. I need to investigate the issu first.
